pkg/webui/lib/components/date-time/index.js

```diff
@@ -63,7 +63,7 @@ class DateTime extends React.PureComponent {
     }
 
     return (
-      <time className={className} dateTime={dateValue.toISOString()} title={result}>
+      <time className={className} dateTime={dateValue.toISOString()} title={noTitle ? ' ' : result}>
```

Comment:
```suggestion
      <time className={className} dateTime={dateValue.toISOString()} title={noTitle ? undefined : result}>
```

Should work for HTML attributes as well iirc.